cert/cert.proto

```diff
@@ -24,4 +24,14 @@ message RawNebulaCertificateDetails {
 
     // sha-256 of the issuer certificate, if this field is blank the cert is self-signed
     bytes Issuer = 9;
-}
\ No newline at end of file
+}
+
+message RawNebulaEncryptedData {
+	RawNebulaEncryptedDataMetadata EncryptionMetadata = 1;
+	bytes Ciphertext = 2;
+}
+
+message RawNebulaEncryptedDataMetadata {
+	string EncryptionAlgorithm = 1;
+	string KeyDerivationParameters = 2;
```

Comment:
I kinda waffled on whether to store parameters & the KDF algorithm separately. By storing them together, I am able to reuse the format seen in the Argon2 reference implementation, PHP's password_hash, etc.